lib/mnesia/src/mnesia.erl

```diff
@@ -278,7 +278,7 @@ stop() ->
 	Other -> Other
     end.
 
--spec change_config(Config::atom(), Value::_) -> ok | {error, term()}.
+-spec change_config(Config::atom(), Value::_) -> {ok, _} | {error, term()}.
```

Comment:
Since you are changing the spec, perhaps it may be a good idea to make it more specific and in line with the documentation.  For example, `Config` is not any `atom()` but two specific ones, and the two `_`s above can also be refined.